### ISSUE TITLE
Format MultiNodeTest failures to show up as test failures in TeamCity

### ISSUE THREAD

**Issue by Aaronontheweb** [akkadotnet/akka.net#1105]
If we could format the failures of multi-node tests such that they show up as proper test failures in TeamCity, that would make it a lot easier to figure out what is and what isn't working.

Any ideas on how to do this?

CC @annymsMthd

**Comment by annymsMthd** [comment]
I had done this in the branch where I did the tests with the app domain
stuff. I modified the console writer sink. I'll see if the branch is still
on our fork.

**Comment by Aaronontheweb** [comment]
CC @annymsMthd any chance we could see a PR for this? Would help a lot with some of the Akka.Cluster stuff that we're doing :D

**Comment by Aaronontheweb** [comment]
cc @akkadotnet/contributors this PR is increasing in urgency - it's really tough to tell what's happening with some of the "racier" specs inside the MultiNodeTest suite right now. Can someone take this?

**Comment by schatekar** [comment]
I have done teamcity integration of custom build scripts before. It is not difficult. I can give it a try but I am middle of making a multinode test pass. I am actually blocked on that. 

The only challenge on this one could be that teamcity expects the messages to be written to standard output. In our case the message are coming from multiple child processes so not sure if teamcity would pick that up automatically. Needs to be tried. [This wiki entry](https://confluence.jetbrains.com/plugins/servlet/mobile#content/view/41485653) explains how to do it.

**Comment by Aaronontheweb** [comment]
The main MN test runner writes to standard out on behalf of all children.

On Tue, Sep 1, 2015 at 3:53 AM, Suhas Chatekar <notifications@github.com>
wrote:

> I have done teamcity integration of custom build scripts before. It is not
> difficult. I can give it a try but I am middle of making a multinode test
> pass. I am actually blocked on that.
>
> The only challenge on this one could be that teamcity expects the messages
> to be written to standard output. In our case the message are coming from
> multiple child processes so not sure if teamcity would pick that up
> automatically. Needs to be tried. This wiki entry
> <https://confluence.jetbrains.com/plugins/servlet/mobile#content/view/41485653>
> explains how to do it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/akkadotnet/akka.net/issues/1105#issuecomment-136673674>
> .
>



-- 
Aaron Stannard,
Santa Monica, CA
p: 858-245-0314
e: aaron@aaronstannard.com
t: @Aaronontheweb
w: http://www.aaronstannard.com/
w: http://captainobvio.us/

**Comment by schatekar** [comment]
@Aaronontheweb I have managed to run the build on my TC project. One of the multinode tests is failing and I see the following on TC output - tells which tests is failing. So I am slightly confused as to what is it that we want changed?

![image](https://cloud.githubusercontent.com/assets/998799/9694261/df7e6432-534e-11e5-97de-c09ccbe3574e.png)

**Comment by Aaronontheweb** [comment]
@schatekar that's not a multi-node spec - need that type of failure message for a full-blown multi-node test.

**Comment by Aaronontheweb** [comment]
Removed this from the 1.1 milestone - not a priority right now but would love to see it implemented!

**Comment by Aaronontheweb** [comment]
We actually did accomplish this in 2017, but we moved off of TeamCity in 2019.
